app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  before_save :update_comment_counter
+
+  def update_comment_counter
+    post = Post.find_by_id(post_id)
+    post.comment_counter = post.comment_counter.to_i + 1
+    post.save
+  end
```

Comment:
- [ ] The same suggestions given to post model, apply to this as well
- Make the method private since it's being called in the callback
- It would be a great idea to use `after_save` callback